Equilobe.DailyReport.Models/Storage/AtlassianUser.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Equilobe.DailyReport.Models.Storage
+{
+    public class AtlassianUser
+    {
```

Comment:
This is not enough. You need to connect the AtlassianUser to a jira instance. Create a many to many map (because one user can be part of multiple instances)

Reply:
Or better, let's not make the many to many map. Let's just use InstanceId as a foreign key. Because if there is another jira instance, it will be a different app client. So I think it is better this way. And if the AtlassianUsers get duplicated, it should be no problem. But we need to link it to the InstalledInstance entity.

Reply:
Ok. I added a foreign key.

Reply:
Ok. I think UserImage is obsolete in this case, now that we have the AtlassianUser that has the same things. Maybe we can phase that out in a different task/PR

Reply:
Sure. I created a new task in Jira for this